.eslintrc.json

```diff
@@ -3,21 +3,21 @@
         "es2021": true,
         "node": true
     },
-    "extends": [
-        "airbnb-base"
-    ],
+    "extends": ["airbnb-base"],
     "parser": "@typescript-eslint/parser",
     "parserOptions": {
         "ecmaVersion": "latest",
         "sourceType": "module"
     },
-    "plugins": [
-        "@typescript-eslint"
-    ],
+    "plugins": ["@typescript-eslint"],
     "rules": {
+        "newline-before-return": "off",
         "import/no-unresolved": "off",
         "import/extensions": "off",
         "class-methods-use-this": "off",
-        "no-console": "off"
+        "no-console": "off",
+        "linebreak-style": "error",
+        "quotes": "off",
```

Comment:
why disable quotes enforcing?

Reply:
eslint wants single quotes for imports
editor wants double quoted for imports

I get errors either way

Reply:
sadge okay